internal/cow/cow.go

```diff
@@ -80,4 +80,6 @@ type Container interface {
 	// container to be terminated by some error condition (including calling
 	// Close).
 	Wait() error
+	// Modify this container
+	Modify(ctx context.Context, config interface{}) error
```

Comment:
I see we're adding this here, but not the implementation. Is it implemented somewhere else?

Reply:
The concrete type of xenon containers is `internal/gcs/container.go:Container` and that struct already has a Modify method but since the interface didn't have that method it wasn't possible to call it so I added it here.